### ISSUE TITLE
Does it work with Elm 0.17?

### ISSUE THREAD

**Issue by chalmagean** [jdudek/elm-clipboard#1]
Hi @jdudek, does this work with the new version fo Elm?

-- Cezar

**Comment by jdudek** [comment]
Hi Cezar!

Thanks for checking out the library. I haven’t looked into Elm 0.17 yet, so I guess it won’t work. I’ll try to find some time in the next few days to take a look. In the meantime, you can try using the original library directly: once you set it up and add appropriate attribute, it’s not going to interfere with the Elm code. This will be sufficient if you don’t need any advanced features, like running a callback after copying—for such advanced cases the Elm library is required.

**Comment by chalmagean** [comment]
Yeah that's what I'm currently doing, it's not ideal but hey... it works :)
